controls/raddiagram/extensions/settingspane/settingspaneview_properties.md

```diff
@@ -0,0 +1,191 @@
+									<extensions:SettingsPaneHomeControl/>
+								</telerik:RadTabItem>
+								<telerik:RadTabItem
+										Header="{telerik:LocalizableResource Key=SettingsPane_SizeTab}"
+										Padding="0"
+										extensions:SettingsPaneView.ContainerEditProperty="IsEnabled"
+										extensions:SettingsPaneView.ContainerEditItemType="Shapes"
+										Margin="0 4 0 0">
+									<extensions:SettingsPaneSizeControl/>
+								</telerik:RadTabItem>
+								<telerik:RadTabItem Header="{telerik:LocalizableResource Key=SettingsPane_StyleTab}" Padding="0" Margin="0 4 0 0">
+									<extensions:SettingsPaneStyleControl x:Name="SettingsPaneStyleControl"/>
+								</telerik:RadTabItem>
+								<telerik:RadTabItem Header="{telerik:LocalizableResource Key=SettingsPane_TextTab}" Padding="0" Margin="0 4 4 0">
+									<extensions:SettingsPaneTextControl/>
+								</telerik:RadTabItem>
+								<telerik:RadTabItem Header="Custom Tab" 
+												   extensions:SettingsPaneView.EditorPropertyName="DataContext.IsVisible"
+													extensions:SettingsPaneView.EditorItemType="Shapes,Custom"
+													extensions:SettingsPaneView.EditorValue="{Binding Path=Visibility,Mode=TwoWay, RelativeSource={RelativeSource Self}}"
+														/>
+								<telerik:RadTabControl.AdditionalContent>
+									<telerik:RadToggleButton
+											FontFamily="{StaticResource TelerikWebUI}"
+											Content="{StaticResource GlyphClose}"
+											FontSize="16"
+											FontWeight="Normal"
+											FontStyle="Normal"
+											IsChecked="{Binding IsActive, Mode=TwoWay, Converter={StaticResource InvertedBooleanConverter}, RelativeSource={RelativeSource TemplatedParent}}"
+											IsBackgroundVisible="False"
+											BorderThickness="0"
+											Foreground="{telerik1:Office2016Resource ResourceKey=MarkerInvertedBrush}"
+											Focusable="False"
+											Margin="2 2 4 2"/>
+								</telerik:RadTabControl.AdditionalContent>
+							</telerik:RadTabControl>
+						</Border>
+					</ControlTemplate>
+				</Setter.Value>
+			</Setter>
+		</Style>
+	</ResourceDictionary> 
+{{endregion}}
+
+To bind the Visibility property of the RadTabItem to a property of your Model you can use the __EditorItemType, EditorPropertyName, and EditorValue__ attached properties. The DataContext of the attached properties in the currently selected RadDiagramShape. The business object is held in the DataContext property. Using this information the EditorPropertyName attached property will be bound to the custom Visibility property. The __EditorItemType__ will specify to which elements to apply this functionality. In this example, we will set it to regular shapes and custom ones. Now one attached property left: EditorValue. As the DataContext is the RadDiagramShape, we can use RelativeSource Self-binding to bind to the __RadTabItem__ Visibility property. __Example 5__ sum up this.
```

Comment:
We can rephrase the last few sentences to make them more formal:

As for the final attached property - EditorValue, since the DataContext is the RadDiagramShape, we can use RelativeSource Self-binding to bind to the __RadTabItem__ Visibility property. __Example 5__ demonstrates this approach.